app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<% if current_user %>
+<%= form_for [topic, post, comment], remote: true do |f| %>
```

Comment:
I was trying different things so I tried this, but it didn't make a difference. I'm changing it back